src/containers/HomePage/inputPanel.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from 'react';
+
+import {Button, Input} from '@fluentui/react-northstar';
+
+const InputPanel = () => {
+    const [teamsChannelUrl, setTeamsChannelUrl] = useState('');
+    const [mattermostChannelUrl, setMattermostChannelUrl] = useState('');
+
+    // Error states
+    const [teamsChannelUrlError, setTeamsChannelUrlError] = useState(false);
+    const [mattermostChannelUrlError, setMattermostChannelUrlError] = useState(false);
```

Comment:
```suggestion
   type FormConfig {
      placeholder: string;
      label: string;
      value: string;
      error?: string | boolean;
      isRequired?: boolean;
   }
   
    enum ConnectFormFields {
     teamsChannelUrl = 'teamsChannelUrl'
     mattermostChannelURL = 'mattermostChannelURL'
    }
    
    const connectFormData: Record<ConnectFormFields, FormConfig> = {
    [ConnectFormFields.teamsChannelURL]: {
      ...
      }
    }
    
   const [connectForm, setConnectForm] = useState<Record<ConnectFormFields, Form>>({...connectFormData});
```
create a common form data structure